### ISSUE TITLE
catalogs vs collections in ep 5

### ISSUE THREAD

**Issue by rbavery** [carpentries-incubator/geospatial-python#143]
the new earth search endpoint (incorrectly?) says there are 7 catalogs, when I think they mean 7 collections. So we should refer to collections throughout, and the earth search v1 STAC API as hosting a single catalog. is that right @vincentsarago?

**Comment by vincentsarago** [comment]
Yep, I think stac-browser is messing with name catalog/collections 

The stac-api has 7 **collections**:

```bash
curl https://earth-search.aws.element84.com/v1/collections | jq '.collections | length'  
7
```
